### ISSUE TITLE
Improve VERS spec

### ISSUE THREAD

**Issue by pombredanne** [package-url/vers-spec#19]
As a follow up to merging the version range spec, there are several items that need attention.
This issue is a list of pointers to these items:

@matt-phylum comments
 - https://github.com/package-url/purl-spec/pull/139#discussion_r1388137731
 - https://github.com/package-url/purl-spec/pull/139#discussion_r1388108223
 - https://github.com/package-url/purl-spec/pull/139#discussion_r1388094695
 - https://github.com/package-url/purl-spec/pull/139#discussion_r1388085869
 - https://github.com/package-url/purl-spec/pull/139#discussion_r1388037663
 - https://github.com/package-url/purl-spec/pull/139#discussion_r1388023200
 - https://github.com/package-url/purl-spec/pull/139#discussion_r1388094695

from @tschmidtb51

- https://github.com/package-url/vers-spec/issues/8
- https://github.com/package-url/purl-spec/pull/139#discussion_r1387815630
- https://github.com/package-url/purl-spec/pull/139#discussion_r1387813300
- https://github.com/package-url/purl-spec/pull/139#discussion_r1387794269

**Comment by mjherzog** [comment]
@pombredanne I cannot figure out how to turn this set of comment references into something useful since the links are hard to follow and https://github.com/package-url/purl-spec/pull/139 was merged almost 2 years ago (in September 2024).
I think that the comments on PR 139 will be very hard to track against the current VERS documentation.
Is there something specific that we need to do here or can we close and wait for questions/comments that are still relevant to be raised during the VERS standard review cycle?

**Comment by mjherzog** [comment]
I suppose that we can reserve some time to review these comments against the final draft for submission to Ecma.

**Comment by mjherzog** [comment]
I have reviewed the comments and do not see any that require changes to our draft VERS specification Clause 5 or the current how-to-parse document.
Removing from the ECMA-VERS 1st Edition project.
